### PR TITLE
Create Plugin: add readme best practice example

### PR DESCRIPTION
**Pull request by sympatheticmoose**
**What this PR does / why we need it**:

Developers contributing plugins often ask for advice on how they can best advertise their plugin to users and what sort of content to include in a README, which in effect becomes the front page for the plugin in the catalog. This PR introduces guidance in the form of comments in the generated README. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

Note - there may be an issue that Grafana.com displays HTML comments. We may want to hold off on this enhancement until investigating further and either fixing or introducing new validation.

### CODE REVIEW

File: packages/create-plugin/templates/common/src/README.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+
+Badges convey useful information at a glance for users whether in the Catalog or viewing the source code. You can use [Shields.io](https://shields.io/badges/dynamic-json-badge) together with the Grafana.com API 
+to create dynamic badges that update automatically when you publish a new version to the marketplace.
+
+- For the logo field use 'grafana'
+- Examples (label: query)
+  - Downloads: $.downloads
+  - Catalog Version: $.version
+  - Grafana Dependency: $.grafanaDependency
+  - Signature Type: $.versionSignatureType
```
Comment (jackw) [inline packages/create-plugin/templates/common/src/README.md:26]: This is neat but I couldn't figure out what I actually needed to do to get the badge url. An example might make it much easier for people to make use of?

https://img.shields.io/badge/dynamic/json?logo=grafana&query=$.version&url=https://grafana.com/api/plugins/grafana-polystat-panel&label=Marketplace&prefix=v&color=F47A20

File: packages/create-plugin/templates/common/src/README.md [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+
+Consider other [badges](https://shields.io/badges) as you feel appropriate for your project.
+
+## Overview / Introduction
+Provide one or more paragraphs as an introduction to your plugin to help users understand why they should use it.  
+
+Consider including screenshots both in [plugin.json](https://grafana.com/docs/grafana/latest/developers/plugins/metadata/#info) and within the README.
```
Comment (jackw) [inline packages/create-plugin/templates/common/src/README.md:33]: Should we point out that plugin.json screenshots should be relative links and README image links should be absolute?

File: packages/create-plugin/templates/common/src/README.md [hunk 3]
```diff
@@ -1,5 +1,49 @@
...
+## Getting Started
+Provide a quick start on how to configure and use the plugin.
+
+## Documentation
+If your project has dedicated documentation available for users, provide links here.
+
+## Contributing
+Do you want folks to contribute to the plugin or provide feedback through specific means? If so, tell them how!
+
+## INCLUDE SCREENSHOTS
```
Comment (jackw) [inline packages/create-plugin/templates/common/src/README.md:47]: Is this a duplicate of L33?

File: packages/create-plugin/templates/common/src/README.md [hunk 4]
```diff
@@ -1,5 +1,50 @@
-<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin -->
+<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin. Uncomment and replace the content here before publishing.
```
Comment (josmperez) [inline packages/create-plugin/templates/common/src/README.md:1]:
```suggestion
<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin. Uncomment and replace the content here before publishing. For help in following Grafana's style recommendations for technical documentation, refer to https://grafana.com/docs/writers-toolkit/.
```

**Review by josmperez — APPROVED** [review]
Suggestion. Overall, LGTM.

**Review by xnyo — APPROVED** [review]
LGTM great work!

File: packages/create-plugin/templates/common/src/README.md (hunk 4 above)
Comment (sympatheticmoose) [reply packages/create-plugin/templates/common/src/README.md:1]: Thanks - added alongside documentation section below.

**Review by jackw — APPROVED** [review]
LGTM! 🚀

**Pull request merged**